src/aio/usock_posix.inc

```diff
@@ -832,6 +838,13 @@ error:
         case NN_USOCK_SRC_FD:
             return;
 
+        case NN_FSM_ACTION:
+            switch (type) {
+            case NN_USOCK_ACTION_ERROR:
+                return;
```

Comment:
This raises a flag when reading the code; perhaps a comment, or even better `nn_assert()`, would help by explaining why an error condition results in a no-op?